appIcons.js

```diff
@@ -162,7 +162,7 @@ var DockAbstractAppIcon = GObject.registerClass({
         this.connect('notify::urgent', () => {
             const icon = this.icon._iconBin;
             this._signalsHandler.removeWithLabel(Labels.URGENT_WINDOWS)
-            if (this.urgent) {
+            if (this.urgent && Docking.DockManager.settings.danceUrgentApplications) {
                 icon.set_pivot_point(0.5, 0.5);
                 this.iconAnimator.addAnimation(icon, 'dance');
```

Comment:
```suggestion
            Mh, I feel it would be better to do something like:

```suggestion
            if (this.urgent) {
                if (Docking.DockManager.settings.danceUrgentApplications) {
                    icon.set_pivot_point(0.5, 0.5);
                    this.iconAnimator.addAnimation(icon, 'dance');
                }
```

Otherwise we might break other logic that depends on urgent windows state.

Reply:
Thanks for the review. I don't know what it'd break, but that seems to work the same, so alright